bciers/apps/administration/app/data/jsonSchema/operationInformation/administrationRegistrationInformation.ts

```diff
@@ -11,13 +12,13 @@ export const createAdministrationRegistrationInformationSchema = async (
   const regulatedProducts: { id: number; name: string }[] =
     await getRegulatedProducts();
 
-  const isRegulatedProducts =
-    !registrationPurposesValue?.includes(
-      RegistrationPurposes.ELECTRICITY_IMPORT_OPERATION,
-    ) &&
-    !registrationPurposesValue?.includes(
-      RegistrationPurposes.POTENTIAL_REPORTING_OPERATION,
-    );
+  const isRegulatedProducts = registrationPurposesValue?.includes(
```

Comment:
This was wrong before?

Reply:
I must admit that I'm not quite sure about this part I borrowed from Andrea. 😄 @andrea-williams, do you have the answer?

Reply:
'twas indeed wrong. Only OBPS Regulated Operations will have regulated products

Reply:
we have a ticket in our backlog to make the UI fix for that